ui/start_ui.py

```diff
@@ -674,10 +758,14 @@ def shutdown_deploy(self):
         serve.shutdown()
 
     def get_ray_cluster(self):
-        command = "conda activate " + self.conda_env_name + "; ray status"
+        command = "source ~/anaconda3/bin/activate; conda activate " + self.conda_env_name + "; ray status"
```

Comment:
We can't assume this path works in other users' environment.

Reply:
Sorry, Forgot to remove that one.

Reply:
fixed